apps/ejabberd/src/mod_muc_room.erl

```diff
@@ -3542,6 +3586,13 @@ set_xoption([{<<"muc#roomconfig_maxusers">>, [Val]} | Opts], Config) ->
     _ ->
         ?SET_NAT_XOPT(max_users, Val)
     end;
+set_xoption([{<<"muc#roomconfig_getmemberlist">>, Val} | Opts], Config) ->
+    case Val of
+        <<"none">> ->
+            ?SET_XOPT(maygetmemberlist, []);
+        _ ->
+            ?SET_XOPT(maygetmemberlist, [binary_to_atom(V, latin1) || V <- Val])
```

Comment:
`binary_to_atom` is very dangerous if we convert any input data from any user to an atom. It would be safer to use `binary_to_existing_atom` provided all acceptable atoms are created (for by typing them in source code). There is good chance they are already created.